### PR TITLE
DRY up some duplication that Code Climate detected

### PR DESCRIPTION
**Pull request by xoebus**
Extracted out two methods to fix some warnings that Code Climate had about duplication.

### CODE REVIEW

File: lib/cli.rb [hunk 1]
```diff
@@ -100,5 +88,15 @@ def submission_url(response_body, host)
       "#{host}/user/#{body['language']}/#{body['exercise']}" 
     end
 
+    def api_feedback(assignments)
```
Comment (xoebus) [inline lib/cli.rb]: I'm not jazzed about this method name but I'm having trouble thinking of a good one.

Comment (kytrinyx) [inline lib/cli.rb]: What about something as mundane as `report(assignments)`?

**Comment by kytrinyx** [comment]
Thanks for doing this! It's so much better.

**Comment by xoebus** [comment]
Re-pushed: I've renamed the method to `report` as you suggested. Thanks!

**Comment by kytrinyx** [comment]
Thank you so much for this!

**Comment by kytrinyx** [comment]
Code Climate went from 3.2 to 3.7 :)

**Pull request merged**